presto-main/src/main/java/com/facebook/presto/server/ResourceManagerModule.java

```diff
@@ -87,9 +85,7 @@ protected void setup(Binder binder)
 
         //TODO: Is it really needed here?
         binder.install(new AnalyzerModule());
-        binder.bind(AnalyzerProvider.class).in(Scopes.SINGLETON);
-        binder.bind(BuiltInQueryPreparer.class).in(Scopes.SINGLETON);
-        binder.bind(NativeQueryPreparer.class).in(Scopes.SINGLETON);
+        binder.install(new CruxAnalyzerModule());
```

Comment:
curious, why this is needed for Resource Manager?

Reply:
It's not needed. Thats why I had a TODO as well. However earlier version of resource group module were relying on some analyzer instance. I wonder why it was there.

I will remove it, let me know if you see any concerns. cc: @swapsmagic

Reply:
let's remove it since it is not needed.